src/Wizard.js

```diff
@@ -318,6 +318,11 @@ export default class Wizard extends Webform {
     });
   }
 
+  checkData(data, flags) {
```

Comment:
This seems like it will cause some problems. I will test it out before including it in a new build.